DigitalLearningSolutions.Data/Models/LearningLogEntry.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace DigitalLearningSolutions.Data.Models
+{
+    using System;
+
+    public class LearningLogEntry : BaseSearchableItem

```

Comment:
Why does this need to be a SearchableItem?

Reply:
The view model's SortItems method requires the object passed in to inherit BaseSearchableItem.